Adhan.d.ts

```diff
@@ -16,7 +16,7 @@ export class PrayerTimes {
 }
 
 export class CalculationParameters {
-  constructor(fajrAngle: number, ishaAngle: number, ishaInterval: number, methodName?: string)
+  constructor(methodName: string|undefined|null, fajrAngle: number, ishaAngle: number, ishaInterval: number, maghribAngle: number)
```

Comment:
Great catch, thank you! We will still need to add the typing for the new PolarCircleResolution type and the accompanying variable on the CalculationParameters type.